omero/users/cli/containers-annotations.txt

```diff
@@ -0,0 +1,63 @@
+Creating containers and annotations
+-----------------------------------
+
+The :omerocmd:`obj` command allows users to create and update OMERO objects,
+this can be used to create containers, i.e. projects, datasets and screens. It
+can also be used to create annotations, and, combined with the
+:omerocmd:`upload` command, file annotations. These annotations can then be
+attached to containers or imported images and plates. This page gives a few
+examples of some simple but fairly common workflows.
+
+Creating containers
+^^^^^^^^^^^^^^^^^^^
+
+Creating a dataset with a name::
+
+   $ bin/omero obj new Dataset name=NewDVSet
+   Dataset:51
+
+And then updating that dataset to add a description::
+
+   $ bin/omero omero obj update Dataset:51 description="A dataset for new DV images"
+   Dataset:51
+
+Creating a screen with a name and description::
+
+	$ bin/omero obj new Screen name=Screen001 description="A short description"
+
+To create a project/dataset hierarchy a link must be created between the two
+containers::
+
+   $ bin/omero obj new Project name=NewImages
+   Project:101
+   $ bin/omero obj new ProjectDatasetLink parent=Project:101 child=Dataset:51
+   ProjectDatasetLink:221
+
+If you are confortable using the command line then you can capture the command
+outputs to feed in to other commands, for example::
+
+	$ dataset=$(bin/omero obj new Dataset name=dataset-1)
+	$ project=$(bin/omero obj new Project name=project-1)
+	$ bin/omero obj new ProjectDatasetLink parent=$project child=$dataset
+	ProjectDatasetLink:222
+
+Creating and attaching annotations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a comment annotation and attach it to a dataset::
+
+	$ bin/omero omero obj new CommentAnnotation textValue="Hello World\!"
```

Comment:
Hmm, bash complained but because I was using double quotes. I'll switch to single quotes and let the end user worry about any escaping they might need to do.